include/deal.II/lac/utilities.h

```diff
@@ -50,6 +51,26 @@ namespace Utilities
                                       const VectorType &v0,
                                       const unsigned int k,
                                       VectorMemory<VectorType> &vector_memory);
+
+    /**
+     * Apply Chebyshev filter of @p op with a given positive @p degree
+     * to the input subspace @p x . @p a and @p b define the unwanted
+     * spectrum $ [a,b] $. @p vector_memory is used to allocate memory for
+     * temporary objects. Chebyshev polynomial will be scaled to unity at
+     * @p a_L to prevent overflow. If @p a_L equals to
+     * <code>numbers::signaling_nan<double>()</code>, no scaling is performed.
```

Comment:
This documentation is too terse. I think that to begin with, few people will understand what a Chebyshev filter is, but you also mention "degree", "unwanted spectrum" and a point "a_L" that are terms you can probably only understand if you already know what this thing is doing.

Can you be more explicit in the documentation what the filter does, and maybe provide a formula or two if that is helpful? For example, I suppose that the filter is an operator that works on the given `x`, and suppose that it is a polynomial of `op`, but that by itself would be useful to state.